omero/sysadmins/version-requirements.txt

```diff
@@ -149,7 +149,7 @@ The planned changes for 5.2 depend upon our support of
   * [5.2] 3.4 dropped, 3.5 deprecated, 3.6 not supported
```

Comment:
I was under the impression 3.4 had already been removed from downloads and testing but I defer to @sbesson & @joshmoore